### PR TITLE
Styling the footer

### PR DESCRIPTION
**Pull request by andrewoh531**

### CODE REVIEW

File: docs/src/components/footer.js [hunk 1]
```diff
@@ -1,55 +1,106 @@
...
   margin-bottom: 0;
+  list-style-type: none;
 `
 
+const UnorderedList = ({ list }) => (
+  <Box width={[1, 1 / 4]}>
+    <Ul>
+      {list.map(i => (
+        <li key={i.text}>
+          <External blank href={i.target}>
```
Comment (jonathaningram) [inline docs/src/components/footer.js]: I think we can leave `blank` off - user can open in new window if they want.

File: docs/src/components/footer.js [hunk 2]
```diff
@@ -1,55 +1,106 @@
...
+            {i.text}
+          </External>
+        </li>
+      ))}
+    </Ul>
+  </Box>
+)
+
+const firstColumn = [
+  { text: 'Support', target: 'https://notify.gov.au/support' },
```
Comment (jonathaningram) [inline docs/src/components/footer.js]: `target` -> `href`

File: docs/src/components/footer.js (hunk 1 above)
Comment (andrewoh531) [reply docs/src/components/footer.js]: Removed

File: docs/src/components/footer.js (hunk 2 above)
Comment (andrewoh531) [reply docs/src/components/footer.js]: Changed

File: docs/src/components/footer.js [hunk 3]
```diff
@@ -1,55 +1,104 @@
...
+
+  justify-content: space-between;
   align-items: start;
+  border-bottom: 1px solid #bfc1c3;
+  padding-bottom: 4rem;
 
-  & > * + * {
-    margin-left: 4rem;
-  }
+  ${space};
```
Comment (jonathaningram) [inline docs/src/components/footer.js]: `<Flex/>` has space props already so you don't need this, will share code sample offline.

**Pull request merged**